boot.go

```diff
@@ -43,10 +47,51 @@ func main() {
 					pkglog.Err("getting config for %s [%s]", serverConfAppName, err)
 					os.Exit(1)
 				}
-				pkglog.Info("starting fetcher on port %d", cnf.FetcherPort)
-				go fetcher.Serve(cnf.FetcherPort)
-				pkglog.Info("starting SSH server on %s:%d", cnf.SSHHostIP, cnf.SSHHostPort)
-				os.Exit(pkg.Run(cnf.SSHHostIP, cnf.SSHHostPort, "boot"))
+				circ := sshd.NewCircuit()
+
+				s3Client, err := storage.GetClient(cnf.HealthSrvTestStorageRegion)
+				if err != nil {
+					log.Printf("Error getting s3 client (%s)", err)
+					os.Exit(1)
+				}
+				kubeClient, err := kcl.NewInCluster()
+				if err != nil {
+					log.Printf("Error getting kubernetes client [%s]", err)
+					os.Exit(1)
+				}
+				log.Printf("Starting health check server on port %d", cnf.HealthSrvPort)
+				healthSrvCh := make(chan error)
+				go func() {
+					if err := healthsrv.Start(cnf.HealthSrvPort, kubeClient.Namespaces(), s3Client, circ); err != nil {
+						healthSrvCh <- err
```

Comment:
Do we have to check if namepsace is present or not ?  builder won't start if namespace is not present or k8s cluster is not working